src/services/lnd/onchain-service.ts

```diff
@@ -208,7 +238,10 @@ export const OnChainService = (
       getBalance,
       getPendingBalance,
       listIncomingTransactions,
+      listOutgoingTransactions,
```

Comment:
Why add this new export?

Reply:
this is part of the second PR, the reason is because this is a split